invisible_cities/evm/histos_test.py

```diff
@@ -57,7 +58,7 @@ def filled_histograms(draw, dimension=0, fixed_bins=None):
     else:
         bins = draw(bins_arrays(dimension=dimension))
 
-    title  = draw(text(characters, min_size=1))
+    title  = draw(text(letters, min_size=1)) + draw(text(characters, min_size=1))
```

Comment:
Instead of defining `title` in multiple places, why not define it once as a strategy, and reuse it multiple times?